### PR TITLE
34206 news landing

### PR DESCRIPTION
**Pull request by Sam152**

### CODE REVIEW

**Comment by Sam152** [comment]
test this please

**Comment by Sam152** [comment]
@benjy, this fails because of:

file_entity cloned from https://github.com/benjy/file_entity.git.           [ok]
Unable to checkout revision d7b92128068f3879d3a3a7a1d2b72228f25df026     [error]

**Comment by benjy** [comment]
yeah known issue, berdir asked me to rebase my branch and I did. It's fixed in #692 although hoping to get it all merged upstream today

**Comment by Sam152** [comment]
test this please

File: agov/modules/custom/agov_article/config/install/views.view.news.yml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        description: ''
+        parent: ''
+        weight: 1
+        context: '0'
+        menu_name: main
+    cache_metadata:
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url.query_args
```
Comment (benjy) [inline agov/modules/custom/agov_article/config/install/views.view.news.yml]: Interesting that this context isn't quoted but the others are?

**Comment by nickschuch** [comment]
test this please

File: agov/modules/custom/agov_article/config/install/views.view.news.yml [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+        - 'user.node_grants:view'
+        - user.permissions
+      cacheable: false
```
Comment (benjy) [inline agov/modules/custom/agov_article/config/install/views.view.news.yml]: This views page should be cacheable.

**Comment by Sam152** [comment]
Fresh export has been added. The caching keys did change, but it's odd max-age is -1 even though the UI says "tag based". Relevant part:

     cache_metadata:
    +  max-age: -1
       contexts:
         - 'languages:language_content'
         - 'languages:language_interface'
         - url.query_args
         - 'user.node_grants:view'
         - user.permissions
    -  cacheable: false
    +  tags: {  }

**Comment by benjy** [comment]
Yeah -1 was what we want, that's CACHE_PERMANENT which is expected when you have tag invalidation.

**Pull request merged**